src/main/java/org/maxgamer/rs/command/commands/Warp.java

```diff
@@ -40,23 +40,7 @@ public void execute(Player p, String[] args) throws Exception {
         }
 
         if (args[0].equalsIgnoreCase("add")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "add" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1132)